frontend/src/metabase/components/ExplicitSize/__mocks__/index.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/no-default-export -- deprecated usage
```

Comment:
Add an additional file necessary for mocking `ExplicitSize` in Jest, since previously there was no `ExplicitSize` folder.